core/src/main/scala/mill/util/EitherOps.scala

```diff
@@ -0,0 +1,18 @@
+package mill.util
+
+import scala.collection.generic.CanBuildFrom
+import scala.collection.mutable
+import scala.language.higherKinds
+
+object EitherOps {
+
+  // implementation similar to scala.concurrent.Future#sequence
+  def sequence[A, B, M[X] <: TraversableOnce[X]](in: M[Either[A, B]])(
```

Comment:
Not sure - should we take first error if there is a `Left` in sequence, or should we accumulate all errors

Reply:
First error is fine for now I think